### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by SS1206** [TerrariaCraft/Issues#65]
If I install the mod, it crashes when I start the game and gives no error code, if I take it out of the mods folder it works perfectly. I am using Forge 1.12.2.

**Comment by SS1206** [comment]
Do I need a 64bit version? because funnily enough I just had to install one for terraria itself.

**Comment by Frank-Einstein** [comment]
I had to use slightly different version for some of these plugins. And deactivating Lux in the options prevented the game from crashing every 5 min on my computer. I also removed McCue.

Here's the exact version for every plugins I'm using:
ColoredLux-1.12.2-12.6.29.jar
CubicChunks-1.12.2-0.0.1184.0-SNAPSHOT-all.jar
HammerLib-1.12.2-2.0.6.31.jar
MusicLayer-1.12.2-12.3.21.jar
TerrariaCraft-1.12.2-12.4.25.jar
